systemtest/src/test/java/io/strimzi/systemtest/upgrade/StrimziUpgradeST.java

```diff
@@ -267,7 +267,7 @@ private void performUpgrade(JsonObject testParameters, ExtensionContext extensio
         logPodImages(clusterName);
         changeClusterOperator(testParameters, NAMESPACE, extensionContext);
 
-        if (TestKafkaVersion.containsVersion(getDefaultKafkaVersionPerStrimzi(testParameters.getString("fromVersion")).version())) {
+        if (TestKafkaVersion.supportedVersionsContainsVersion(getDefaultKafkaVersionPerStrimzi(testParameters.getString("fromVersion")).version())) {
```

Comment:
So I assume this change will cause the errors in the CO (about unsupported Kafka version) to be ignored and move directly to the upgrade?

Reply:
Yes, if the Kafka version is unsupported, we will not wait and we'll move directly to upgrade, how you wrote.